_rules/SC1-1-1-image-accessible-name-is-placeholder-text.md

```diff
@@ -0,0 +1,78 @@
+---
+name: Image accessible name is placeholder text
+description: |
+  This rule checks that image elements that use generic placeholder text as their accessible name do so without loss of infomation to the user.
+
+success_criterion:
+- 1.1.1 # Non-text Content
+
+test_aspects:
+- DOM Tree
+- CSS Styling
+
+authors:
+- Bryn Anderson
+---
+
+### Applicability
+
+The rule applies to HTML `input type="image"` or `area` elements, or any HTML element with the semantic role of `img`, where the element is [exposed to assistive technologies](#exposed-to-assistive-technologies) and has an [accessible name](#accessible-name) that exclusively uses [generic placeholder text](#generic-placeholder-text).
```

Comment:
link semantic role to its definition

Reply:
Additionally, this should be applicable only to non-decorative images

Reply:
I know what you mean but non-decorative is considered subjective and their fore cannot be included as part of the applicability as per Wilco's comment on the ACT rules format https://github.com/auto-wcag/auto-wcag/pull/263#discussion_r253140910